cirq-core/cirq/ops/control_values.py

```diff
@@ -206,79 +239,77 @@ class SumOfProducts(AbstractControlValues):
 
         >>> nand_control_values = cirq.SumOfProducts(((0, 0), (0, 1), (1, 0)))
         >>> q0, q1, q2 = cirq.LineQubit.range(3)
-        >>> nan_cop = cirq.X(q2).controlled_by(q0, q1, control_values=nand_control_values)
+        >>> nand_cop = cirq.X(q2).controlled_by(q0, q1, control_values=nand_control_values)
     """
 
-    _internal_representation: Tuple[Tuple[int, ...], ...]
-
-    def __post_init__(self):
-        if not len(self._internal_representation):
-            raise ValueError('SumOfProducts can\'t be empty.')
-        num_qubits = len(self._internal_representation[0])
-        for p in self._internal_representation:
-            if len(p) != num_qubits:
-                raise ValueError(
-                    f'size mismatch between different products of {self._internal_representation}'
-                )
-        if len(self._internal_representation) != len(
-            set(map(tuple, self._internal_representation))
-        ):
-            raise ValueError('SumOfProducts can\'t have duplicate products.')
-
-    def _identifier(self) -> Tuple[Tuple[int, ...], ...]:
-        return self._internal_representation
-
-    def _expand(self) -> Iterator[Tuple[int, ...]]:
+    def __init__(self, data: Sequence[Union[int, Collection[int]]], *, name: Optional[str] = None):
+        self._conjugations: Tuple[Tuple[int, ...], ...] = tuple(
```

Comment:
nit: "conjunctions" rather than "conjugations"
```suggestion
        self._conjunctions: Tuple[Tuple[int, ...], ...] = tuple(
```

Reply:
done